cloud_info_provider/providers/opennebula.py

```diff
@@ -88,7 +88,7 @@ def get_images(self, **kwargs):
         }
         defaults = self.static.get_image_defaults(prefix=True)
         img_schema = defaults.get('image_schema', 'template')
-        group_name = kwargs.get('project', None)
+        group_name = kwargs.get('auth', {}).get('project_id', None)
```

Comment:
Thanks @orviz !
AFAIU here we should have `group` instead of `project_id`, as in the yaml configuration for OpenNebula we expect to have a group under the `auth` section.

Reply:
oops I missed this one.. I'll change project_id -> group